src/dotnet-releaser/ReleaserApp.Publishing.cs

```diff
@@ -32,9 +33,6 @@ public partial class ReleaserApp
                 {
                     var appPackagesToPublish = buildPackageInformation.AppPackages;
 
-                    // In the case of a build, we still want to upload a draft release notes
```

Comment:
But in that case, it doesn't upload anymore the packages here no? That would require to split `devHosting.UpdateChangelogAndUploadPackages`